sycl/doc/GlobalObjectsInRuntime.md

```diff
@@ -88,7 +88,18 @@ constructor and destructor.
 
 ## Plugins
 
-TBD
+Plugin lifetime is managed by utilizing piInitializePlugins() and piTearDown().
+GlobalHandler::shutdown() will tear down all registered globals befoe SYCL RT
+library is unloaded. It will invoke piTearDown() and unload() for each
+plugin. piTearDown() is going to perform any necessary tear-down process at the
+plugin PI level. These two apis allow on-demand plugin lifetime management. SYCL
+RT can control the beginning and the end of the plugin. 
+![](images/plugin-lifetime.png)
+
+Each plugin should not create global variables that require non-trivial
+destructor. Pointer variables with heap memory allocation is a good example
+to be created at the global scope. A std::vector object is not. piTearDown
+will take care of deallocation of these global variables safely.
```

Comment:
Probably this section should go to PluginInterface as well.

Reply:
Thanks. Done.